handcalcs/decorator.py

```diff
@@ -30,9 +28,9 @@ def wrapper(*args, **kwargs):
                         "jupyter_display option requires IPython.display to be installed."
                     )
                 display(Latex(latex_code))
-                return calculated_results
+                return scope.return_value
             latex_code = latex_code.replace("\\[", "").replace("\\]", "")
-            return (left + latex_code + right, calculated_results)
+            return (left + latex_code + right, scope.inner_scope)
```

Comment:
How is the last element in the returned tuple used?  Does it need to be the dict of locals?  Should it also include the globals and closures that were used?  Or, can it be the return value of the function?

Reply:
Well, now that innerscope is accessing the namespaces, the function should just return the return value, whatever the user wants.

Reply:
The namespace dicts are only needed internally to substitute values in the Latex code.

Reply:
Cool, so if you want the return value here too, then change the last item to `scope.return_value`.